components/link/index.css

```diff
@@ -99,13 +99,17 @@
 	--mod-link-text-color-hover: var(--mod-link-text-color-white, var(--spectrum-white));
 	--mod-link-text-color-active: var(--mod-link-text-color-white, var(--spectrum-white));
 	--mod-link-text-color-focus: var(--mod-link-text-color-white, var(--spectrum-white));
+
+	--spectrum-link-focus-indicator-color: var(--spectrum-static-white-focus-indicator-color);
```

Comment:
These weren't explicitly defined in the link token specs. However, it didn't make sense to have a static black link with a blue focus ring. It doesn't really match how most of the other static color components were handling their focus rings.
<img width="213" alt="Screenshot 2025-05-02 at 12 30 14 PM" src="https://github.com/user-attachments/assets/7abfc813-85bd-429f-8fd9-71684a116606" />
<img width="200" alt="Screenshot 2025-05-02 at 12 29 54 PM" src="https://github.com/user-attachments/assets/5d36b9f3-ae89-4ce0-9556-a1aa0f69a8ea" />